src/azure-cli/azure/cli/command_modules/storage/azcopy/util.py

```diff
@@ -74,7 +74,9 @@ def check_version(self):
 
     def run_command(self, args):
         args = [self.executable] + args
-        logger.warning("Azcopy command: %s", args)
+        args_hides = args.copy()
+        args_hides[2] = args_hides[2][0:args_hides[2].index('?')]
+        logger.warning("Azcopy command: %s", args_hides)
```

Comment:
Is there possibility that `args` doesn't contain a sas token?

Better to check if args contains any arg having substring `&sig=` and then we can hide the substrings after `&sig=`